pkg/apis/app/v2/kieapp_types.go

```diff
@@ -138,17 +142,19 @@ type KieServerSet struct {
 	Jms          *KieAppJmsObject `json:"jms,omitempty"`
 	Jvm          *JvmObject       `json:"jvm,omitempty"`
 	// JbpmCluster Enable the KIE Server Jbpm clustering for processes fail-over, it could increase the number of kieservers
-	JbpmCluster bool            `json:"jbpmCluster,omitempty"`
-	Kafka       *KafkaExtObject `json:"kafka,omitempty"`
+	JbpmCluster     bool            `json:"jbpmCluster,omitempty"`
+	Kafka           *KafkaExtObject `json:"kafka,omitempty"`
+	StartupStrategy string          `json:"startupStrategy,omitempty"`
```

Comment:
if already setting at `spec.startupStrategy` ... this and the kieserverset addition shouldn't be required... unless you want the user to be able to set them differently, in which case the spec field should be removed.

that said, the template fields below would stay.